manifests/resource/location.pp

```diff
@@ -269,6 +271,7 @@
   Optional[Enum['any', 'all']] $location_satisfy                   = undef,
   Optional[Array] $location_allow                                  = undef,
   Optional[Array] $location_deny                                   = undef,
+  Optional[String] $location_acl                                   = undef,
```

Comment:
I suppose the name can't be an empty string 
```suggestion
  Optional[String[1]] $location_acl                                = undef,
```

Also maybe the user must be able to overwrite the default path `nginx::conf_dir/conf.d` ?